src/python/python.cpp

```diff
@@ -276,10 +276,10 @@ PYBIND11_MODULE(onnxruntime_genai, m) {
       .def(pybind11::init<Model&, PySearchParams&>())
       .def("IsDone", &PyGenerator::IsDone)
       .def("ComputeLogits", &PyGenerator::ComputeLogits)
```

Comment:
Add a new function: GetLogits?

Reply:
Yep, adding that as part of the other API changes, just wanted to submit this separately.